app/models/saved_claim/veteran_readiness_employment_claim.rb

```diff
@@ -129,6 +126,9 @@ def send_to_vre(user)
           log_exception_to_sentry(e, { uuid: user.uuid })
         end
       end
+    else
+      log_message_to_sentry('Participant id is blank when submitting VRE claim', :warn)
```

Comment:
Does this have to go to Sentry? What about logging instead and setting up a monitor in Datadog for this warning scenario?

Reply:
this is a relocated log. the monitors and dashboards have already been configured around it.

Reply:
Ok, no worries. Moving forward, I would recommend logging + monitors to avoid flooding Sentry with things other than errors